sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_differentiate_output_labeled_tables_async.py

```diff
@@ -0,0 +1,138 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_differentiate_output_labeled_tables_async.py
+
+DESCRIPTION:
+    This sample demonstrates the differences in output that arise when begin_recognize_custom_forms
+    is called with custom models trained with fixed vs. dynamic table tags.
+    The models used in this sample can be created in the sample_train_model_with_labels.py using the
+    training files in
+    https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/formrecognizer/azure-ai-formrecognizer/samples/sample_forms/labeled_tables
+
+    Note that Form Recognizer automatically finds and extracts all tables in your documents whether the tables
```

Comment:
I'm not sure I follow the purpose of this text. Could you expand on the reason why we need this?

Reply:
We've gotten feedback that our design makes the OCR tables hard to find. Didn't want to give the impression with this sample that you must label tables to get tables in the output. The service team noted this in the service docs and I thought it would be important to underscore that here as well.

Reply:
I see. I guess the part that is kind of confusing is 
>Therefore, you don't have to label every table from your form with a table tag and your
    table tags don't have to replicate the structure of every table found in your form.

Maybe something like:

..are tagged/labeled or not. Tables extracted automatically by Form Recognizer will be included in the `tables` property under `RecognizedForm.pages`. If this doesn't satisfy your needs, you can label the table.
A conceptual explanation ....

Reply:
Got it - will update to your suggestion to make it less confusing.